Delete_Exchange_Webshells.py

```diff
@@ -0,0 +1,116 @@
+"""
+"""
```

Comment:
We also should have a top level description and tooltips of the key actions to make this easier to read.

Reply:
Done